3rdParty/fuerte/src/HttpConnection.cpp

```diff
@@ -307,18 +307,23 @@ std::string HttpConnection<ST>::buildRequestBody(Request const& req) {
   } else {
     header.append("Connection: Close\r\n");
   }
+  bool haveAuth = false;
   for (auto const& pair : req.header.meta) {
     if (boost::iequals(fu_content_length_key, pair.first)) {
       continue;  // skip content-length header
     }
 
+    if (pair.first == "Authorization" || pair.first == "authorization") {
```

Comment:
HTTP headers are allowed to have any case, so `aUthOriZAtion` would also be ok :smile:
```suggestion
    if (boost::iequals("authorization", pair.first))
```